addons/project/views/project_task_views.xml

```diff
@@ -580,22 +586,39 @@
                                         <b t-if="record.rating_active.raw_value and record.rating_count.raw_value &gt; 0" groups="project.group_project_rating">
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-smile-o text-success" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-meh-o text-warning" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-frown-o text-danger" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
+                                            <span class="fa fa-fw fa-smile-o text-success rating_face" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
+                                            <span class="fa fa-fw fa-meh-o text-warning rating_face" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
+                                            <span class="fa fa-fw fa-frown-o text-danger rating_face" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
                                         </b>
-                                        <a t-if="record.subtask_count.raw_value and record.allow_subtasks.raw_value" class="subtask_list_button" title="See Subtasks" style="padding: 0px 5px 0px 0px; box-sizing: border-box; font-size: 20px;"/>
+                                        <a t-if="!record.project_id.raw_value" class="text-muted" style="font-size: 17px; padding-top: 1.5px;">
+                                            <i title="Private Task" class="fa fa-lock"/>
+                                        </a>
+                                        <t t-if="!record.is_private.raw_value and record.subtask_count.raw_value">
+                                            <div t-attf-class="{{record.subtask_count.value &gt; record.closed_subtask_count.value}}"></div>
+                                            <a t-if="!record.is_private.raw_value and record.subtask_count.raw_value and record.subtask_count.value &gt; record.closed_subtask_count.value" class="text-muted">
+                                                <span title="See Subtasks" class="subtask_list_button fa fa-check-square-o me-1"/>
+                                                <t t-out="record.closed_subtask_count.value"/>/<t t-out="record.subtask_count.value"/>
+                                            </a>
+                                            <div t-else="" class="text-muted">
+                                                <span title="e" class="fa fa-check-square-o me-1"/>
```

Comment:
I did not see we also have that in 16.2, why do we have `title="e"`? 🤯

Reply:
i will also fix it in 16.2, title should be "subtasks", we get a warning it a span doesn't contain a title attribute

Reply:
Check first with Alexandra to be sure which title we have to put here.